docs/source/upgrade.md

```diff
@@ -2,6 +2,83 @@
 
 This pages describes Scylla Operator upgrade procedures.
 
+## `v1.0.0` -> `v1.1.0`
+
+During this update we will change readyness probes for Scylla Operator, and initContainer of Scylla Pods, and
+container images.
+
+1. Get name of StatefulSet managing Scylla Operator
+   ```shell
+   kubectl --namespace scylla-operator-system get sts --selector="control-plane=controller-manager"
+
+   NAME                                 READY   AGE
+   scylla-operator-controller-manager   1/1     95m
+   ```
+
+1. Change readyness probe and used container image by applying following patch:
```

Comment:
we may not be there yet, but I'd have just expected a line saying to update the operator deployment like `kubectl apply -f ./deploy/scylla-operator/statefulset.yaml` and be done

Reply:
Yeah, it's a consequence of having single manifest with everything.